src/main/java/io/vinyldns/java/model/batch/BatchResponse.java

```diff
@@ -21,10 +21,16 @@
   private String id;
   private String userId;
   private String userName;
-  private String comments;
+  private String comments; //optional
   private Date createdTimestamp;
   private List<SingleChange> changes;
   private BatchChangeStatus status;
+  private String ownerGroupId; //optional
+  private String ownerGroupName; //optional
+
+  public BatchResponse(String ownerGroupId) {
```

Comment:
We shouldn't add a new constructor here, since it overrides the default constructor.

Since `ownerGroupId` is not required, adding a new constructor here means we would have to at least add an empty constructor (which is the default constructor in Java):
```
public BatchResponse() {}
```